pkg/util/strings.go

```diff
@@ -63,3 +63,20 @@ func PrefixStringArray(a []string, prefix string) []string {
 
 	return b
 }
+
+// Diff returns the elements in `compareWhat` that are not in `compareTo`.
+func Diff(compareWhat, compareTo []string) []string {
```

Comment:
Are you sure you want to call it diff? It should return diff from both list, so in both ways

Reply:
changed